app/forms/schools/add_participant_form.rb

```diff
@@ -218,18 +217,18 @@ def check_records
     end
 
     def check_for_existing_profile
-      self.existing_participant_profile = ParticipantProfile::ECF.joins(:ecf_participant_validation_data)
-                                                        .where("LOWER(full_name) = ? AND trn = ? AND date_of_birth = ?",
-                                                               full_name.downcase,
-                                                               formatted_trn,
-                                                               date_of_birth).first
+      self.existing_participant_profile = ParticipantProfile::ECF
+                                            .joins(:ecf_participant_validation_data)
+                                            .where(ecf_participant_validation_data: { trn: formatted_trn })
+                                            .first
```

Comment:
What happens when we have more than one profile with the same TRN?

Reply:
The SIT will be asked if this new participant actually is a school transfer. 
If they answer YES they will be deviated to the transfer journey.
Otherwise they will see a screen pointing them to contact Support